docs/vsphere_ci/scripts/enable-containers-feature.ps1

```diff
@@ -0,0 +1,8 @@
+# Powershell script to enable the optional Containers feature on a Windows Server operating system
+#
+# USAGE
+#    ./enable-containers-feature.ps1
+
+Install-WindowsFeature -Name Containers
+# Toggling Windows feature requires reboot to take effect
+Restart-Computer -Force
```

Comment:
`sysprep` will reboot the VM anyway, no need to force it here. There is no subsequent  command  that depends on this, so we should be good.

Reply:
To make it simpler, you can move the PowerShell cmdlet (`Install-WindowsFeature -Name Containers`) as an [inline command](https://github.com/openshift/windows-machine-config-operator/pull/1052/files#diff-f95d5d0ee98dedf392fb54c0fb93f7e3df3c9699b01d08ca8fd00d30ccba8244R92) as part of the autounattend.xml.

Reply:
Great suggestion, will do. I don't quite understand where `sysprep` is called though. Is this done as part of the packer run?

Reply:
That's correct, it's done as part of the packer process